com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/DefaultSettingsPanel.cs

```diff
@@ -146,30 +154,45 @@ public void OnActivate(string searchContext, VisualElement rootElement)
         {
         }
 
+        void Draw_Warnings(ref SerializedHDRenderPipelineGlobalSettings serialized, Editor owner)
```

Comment:
I think code convetion does not allow '_' inside names

Reply:
The rest of the file is like that and other windows UI code, so it should be good no?

Reply:
Well, according to code convention, we should not use _ except for prefixing https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs

So I assum it was already wrong at the time :/